rust/wasmtime_bindings/src/lib.rs

```diff
@@ -106,7 +107,7 @@ pub struct Instance {
 
 fn create_instance(engine: &Engine, module: &Module, store: &mut Store) -> Result<Box<Instance>> {
     let mut linker = wasmtime::Linker::new(&engine.wasmtime_engine);
-    wasmtime_wasi::add_to_linker(&mut linker, |s| s).context("Failed to add wasi to linker")?;
+    wasmtime_wasi::preview1::add_to_linker_async(&mut linker, |s| s).context("Failed to add wasi to linker")?;
```

Comment:
This is weird, what is this "preview1" thing? It doesn't sound very confidence-inspiring....

Reply:
It refers to the "WASI Snapshot Preview 1". It was added because now there's also "WASI Snapshot Preview 2"

Reply:
So it's not just a snapshot, it's a preview, and it's not even the second preview - it's the first one. Let's build our product on this! :-)